1-exercises/A-reading-list/script.js

```diff
@@ -1,27 +1,39 @@
 function readingList(books) {
-  // Write your code here...
+  let ulEl = document.createElement("ulEl");
+  ulEl.setAttribute("id", "proList");
+  let content = document.querySelector("#content");
+
+  for (book of books) {
+    let liEl = document.createElement("liEl");
+    liEl.setAttribute("class", "item");
```

Comment:
One thing missing li background color per the **alreadyRead** status

Reply:
I don't understand this part. Please could you explain?